project/Build.scala

```diff
@@ -817,6 +817,74 @@ object Build {
       javaOptions := (javaOptions in `dotty-compiler-bootstrapped`).value
     )
 
+  lazy val `dotty-scala-library-from-tasty-tests` = project.in(file("dotty-scala-library-from-tasty-tests")).
+    withCommonSettings(Bootstrapped).
+    dependsOn(`dotty-tasty-inspector` % "test->test").
+    settings(commonBootstrappedSettings).
+    settings(
+      javaOptions := (javaOptions in `dotty-compiler-bootstrapped`).value,
+      javaOptions += "-Ddotty.scala.library=" + packageBin.in(`dotty-scala-library`, Compile).value.getAbsolutePath
+    )
+
+
+  // Scala library compiled by dotty using the latest published sources of the library
```

Comment:
I suggest using the stdlib sources from our community-build instead, that way if we need to change something we can change it in the community build submodule.

Reply:
Those sources are not aligned with the library that we are using to run dotty apps. Their documentation may also slightly defer.

Reply:
We should keep them aligned if possible.

Reply:
If you change things because you need to change something, by definition they're not aligned anymore.

Reply:
Sure but we can make PRs to keep them aligned if needed. I think it's just confusing to operate on multiple versions of scala-library at the same time, I don't see the advantages.

Reply:
Under that logic, if we don't want to operate on multiple versions of the scala-library, we should remove the stdlib213 from the community-build as it is not aligned with the one we use to run dotty. But the one in the community build allows us to future proof the compilation of the library, therefore it is good to have this second variant of the scala-library.